pcf/test/particle/aws/iam/test_iam_role.py

```diff
@@ -0,0 +1,92 @@
+# Copyright 2018 Capital One Services, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import moto
+import boto3
+import json
+
+from pcf.particle.aws.iam.iam_role import IAMRole
+from pcf.core import State
+
+
+class TestIAMRole:
+
+    @moto.mock_iam
+    def test_apply_states(self):
```

Comment:
we should add an additional test for the feature that iam role takes in the iam policies as its parents.